content/en/docs/spinnaker-user-guides/using-dinghy.md

```diff
@@ -79,6 +82,7 @@ Here is an example Dinghyfile:
 Make sure you specify the following fields so that the Dinghyfile can create a pipeline with stages:
 
 * `.application`: The name of the application where pipelines will be created or updated.  If the application does not exist, it will be created.
+* `appmetadata`: Key/value pairs where you can define application metadata, such as whether Canary analysis is enabled (`"canary": "true"`). Note that no validation is done for this field. Make sure that you model it after pipeline JSON that you know works.
```

Comment:
> pipeline JSON

So in this case... I'm not sure how users are going to get the fields they want defined. I don't think there's a clear and obvious way from the UI to get application metadata. Probably the best way to get this information would be to crack open the developer console and print out the value of `spinnaker.application` while in the Application config view.

I recognize what a poor UX that is, but I'm not aware of a better way to do so at the moment :(